static/js/tippyjs.js

```diff
@@ -142,6 +143,7 @@ export function initialize() {
         delay: 0,
         target: "#streams_inline_cog",
         onShow(instance) {
+            popovers.hide_all();
```

Comment:
Do we also need logic to hide Tippy popovers in `popovers.hide_all()`?

Reply:
No, tippy popover hide on their own when you click outside them.

Reply:
We actually needed `hide_all_except_sidebars` here.